library/MedEx/MedEx_background.php

```diff
@@ -0,0 +1,80 @@
+<?php
+/**
```

Comment:
Good point.  It is meant to be the callback method for instant updates as opposed to crontab MedEx or background_services MedEx_background.
I'll add in a check against the API_key for MedEx.php receive/callbacks.